internal/v2/repository/policy/policy_password_lockout.go

```diff
@@ -63,10 +64,29 @@ func (e *PasswordLockoutPolicyChangedEvent) Data() interface{} {
 
 func NewPasswordLockoutPolicyChangedEvent(
 	base *eventstore.BaseEvent,
+	changes []PasswordLockoutPolicyChanges,
 ) *PasswordLockoutPolicyChangedEvent {
-	return &PasswordLockoutPolicyChangedEvent{
+	changeEvent := &PasswordLockoutPolicyChangedEvent{
 		BaseEvent: *base,
 	}
+	for _, change := range changes {
```

Comment:
maybe we should check if length of changes is bigger than 0

Reply:
and return error?

Reply:
changed as discussed